src/components/TransactionPanel/TransactionPanel.js

```diff
@@ -51,7 +53,7 @@ import PanelHeading, {
 } from './PanelHeading';
 
 import css from './TransactionPanel.module.css';
-
+const QRCode = require('qrcode.react');
```

Comment:
@emipasat why not regular import?

Reply:
I've followed the example from the qrcode.react repository and it was like this in there. I will change it to regular import.